tools/ci/checks/get_base_branch.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+set -e
+
+current_branch=${CURRENT_GIT_BRANCH:-$( git symbolic-ref --short HEAD )}
+
+base_branch="master"
+
+if [[ x"$current_branch" == x*"pull/"* ]]; then
+    # This is a PR and we need to determine the branch from the API.
+    pr_name="${current_branch/pull/pulls}"
+
+    if [ -z ${GIT_REPO} ]; then
+        set -x
+        git_repo=$( git remote get-url origin )
```

Comment:
fwiw it is also good practice to quote command substitutions, e.g. `git_repo="$(git remote get-url origin)"`.